utils/scripts/install-centos8.sh

```diff
@@ -67,5 +65,6 @@ dnf install \
     python3-yaml \
     sg3_utils \
     valgrind-devel \
+    which \
```

Comment:
Is this change needed?

Reply:
Just checking it was intentional.  Best to leave it here and have the minimal set in the conditional script.